Update readme nvm windows #5 (#13)
uqbar-project/wollok-ts-cli: README.md

### Node, npm, and dependencies

> **For Linux, MacOS:**

Reviewer: ```suggestion
#### For Linux, MacOS:
```

# This will install all the project dependencies. Give it some time.
npm install
```
**For Windows:**

You need to install the [NVM- for Windows](https://github.com/coreybutler/nvm-windows)

Run the installer `nvm-setup.exe` as Administrator

> Open an elevated Command Prompt or Git Bash (with Administrator privileges) and run

Reviewer: ```suggestion
Open an elevated Command Prompt or Git Bash in the project folder (with Administrator privileges) and run
```

```bash

> nvm install (version)
> nvm use (version)
> #The version that is being used is 16.15.0

Reviewer: ```suggestion
nvm install <<version number>>
nvm use <<version number>>
# The version number is in the .nvmrc file (do not use codename version e.g. lts/gallium, in Windows you have to use the equivalent version number e.g. 16.15.0)
```
